genie-client/src/main/java/com/netflix/genie/client/config/GenieNetworkConfiguration.java

```diff
@@ -30,14 +30,19 @@
 @Setter
 public class GenieNetworkConfiguration {
 
+    /**
+     * Default network timeout value if not specified.
+     */
+    public static final long DEFAULT_TIMEOUT = -1;
```

Comment:
-1 basically means its not set by user so i will let okhttp use defaults. which is believe is 10000.